docs/docs/how-tos/nebari-aws.md

```diff
@@ -94,23 +94,23 @@ installed in your environment, (iii) opted for **AWS** as your cloud provider, a
 installation, please visit the "Getting Started" section of our [troubleshooting page](/troubleshooting) for further guidance.
 :::
 
-## Nebari Initialize
+## Initializing Nebari
 
 Great, you’ve gone through the [Nebari Installation](/getting-started/installing-nebari.md) and [authentication setup](#authentication) steps, and have ensured that all the necessary
-environment variables have been properly set. It is time to initialize and deploy Nebari!
+environment variables have been properly set.
+
+In this step, you'll run `qhub init` to create the `qhub-config.yaml`.
 
 1. In your terminal, start by creating a new project folder. For this demonstration, we will name the new folder `nebari-aws`:
 
    ```bash
    mkdir nebari-aws && cd nebari-aws
    ```
 
-2. Executing the command below will generate a basic config file with an infrastructure based on **AWS**, with project name `projectname`, endpoint domain `domain`, and with the authentication mode set to **password**.
+2. Executing the command below will generate a basic config file with an infrastructure based on **AWS**. The `projectname` will be the name of the folder/repo that will manage this Nebari deployment (it will be created). The `domain` will the domain endpoint for your Nebari instance. The `auth-provider` sets your authentication provider that you plan to use inside of Keycloak, options are Github, Auth0, and password. For this example, we'll run with project name `projectname`, endpoint domain `domain`, and with the authentication mode set to **password**. These can be updated later by directly modifying the `qhub-config.yaml`.
```

Comment:
Suggestion: split into bullets so that the keys in the config.yml file are easier to parse/read

Reply:
done.